doc/source/advanced/datatype.rst

```diff
@@ -0,0 +1,90 @@
+***********************************
+Data Types in Lux
+***********************************
+
+.. note:: You can follow along this tutorial in a Jupyter notebook. 
```

Comment:
Add link to `[Github] [Binder]` here

Reply:
I can add this once the tutorial is merged!

Reply:
I merged the tutorials in now, you can add in the links to Github and Binder